depender/pom.xml

```diff
@@ -14,10 +14,9 @@
 
   <dependencies>
     <dependency>
-      <groupId>${pom.groupId}</groupId>
+      <groupId>${project.groupId}</groupId>
       <artifactId>dependee</artifactId>
-      <version>${pom.version}</version>
-      <optional>true</optional>
```

Comment:
Anyway, -1 for this change, as another test in future might need an optional dependency.
It's better to add a new plugin mandatory-depender.
(I should have named this plugin optional-depender!)